### PR TITLE
[20170] Limit permissions and include missing certificates in security tests

### PR DESCRIPTION
**Pull request by Tempate**
In this version, we have: 
- Generated a security certificate per participant to avoid sharing certificates.
- Restricted the permissions to the strictly necessary.
- Replaced the tests' README files with comments in the compose files (the current standard).

In the process, we discovered a Fast-DDS issue:
- https://github.com/eProsima/Fast-DDS/pull/4188

### CODE REVIEW

File: ddsrouter_test/compose/test_cases/security/backdoor/compose.yml [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+# │topic_1   ├───► ┌────┐ ┌──────┐ ◄───┤topic_0   │
+# └──────────┘   │ │sec │ │no sec│ │   └──────────┘
+#                │ │    │ │      │ │
+# ┌──────────┐   │ │xml │ │local │ │   ┌──────────┐
+# │subscriber├───► └────┘ └──────┘ ◄───┤subscriber│
+# │          │   │                 │   │          │
+# │sec       │   └─────────────────┘   │no_sec    │
+# │          │                         │          │
+# │topic_0   │                         │topic_1   │
+# └──────────┘                         └──────────┘
```
Comment (LuciaEchevarria99) [inline ddsrouter_test/compose/test_cases/security/backdoor/compose.yml:23]: As you delete the README and attempt to schematize the information, please ensure that you do not remove any important details. For example, retain the information in the 'Topics' section and the 'Expected Behavior' section.

File: ddsrouter_test/compose/test_cases/security/backdoor_dds/compose.yml [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
+# │topic_1   ├───► ┌────┐ ┌──────┐ ◄───┤topic_0   │
+# └──────────┘   │ │sec │ │no sec│ │   └──────────┘
+#                │ │    │ │      │ │
+# ┌──────────┐   │ │xml │ │xml   │ │   ┌──────────┐
+# │subscriber├───► └────┘ └──────┘ ◄───┤subscriber│
+# │          │   │                 │   │          │
+# │sec       │   └─────────────────┘   │no_sec    │
+# │          │                         │          │
+# │topic_0   │                         │topic_1   │
+# └──────────┘                         └──────────┘
```
Comment (LuciaEchevarria99) [inline ddsrouter_test/compose/test_cases/security/backdoor_dds/compose.yml:23]: Same as in the other Compose, try not to lose information. Apply in the rest of tests.

**Review by LuciaEchevarria99 — CHANGES_REQUESTED** [review]

File: ddsrouter_test/compose/test_cases/security/backdoor/compose.yml (hunk 1 above)
Comment (Tempate) [reply ddsrouter_test/compose/test_cases/security/backdoor/compose.yml:23]: The information is contained in `Test description`. Read the last sentence.

File: ddsrouter_test/compose/test_cases/security/backdoor_dds/compose.yml (hunk 2 above)
Comment (Tempate) [reply ddsrouter_test/compose/test_cases/security/backdoor_dds/compose.yml:23]: Same reply as to the previous comment.

File: ddsrouter_test/compose/test_cases/security/backdoor/compose.yml (hunk 1 above)
Comment (juanlofer-eprosima) [reply ddsrouter_test/compose/test_cases/security/backdoor/compose.yml:23]: Please rebase.

**Review by LuciaEchevarria99 — APPROVED** [review]
LGTM

**Pull request merged**